src/main.py

```diff
@@ -1,14 +1,21 @@
-import parse_fasta
-import sam_to_fasta_file
+from pathlib import Path
+
+from src import parse_fasta
+from src import sam_to_fasta_file
 
 # this method starts the tasks for the whole exercise (1.4 and 2.2 to be precise)
 
 if __name__ == '__main__':
     print(parse_fasta.map_reads(
-        r"/Users/Aline/Desktop/21HS/21HS PrBi/PrBi Alexander Kanitz/PrBi/RNA-Seq/Test files/sequences.fasta",
```

Comment:
**Never** use absolute paths in public code: apart form the fact that it will never be portable, it's also a security issue, because you are revealing potentially sensitive information about your file system to the public.